src/documentation/reference/ReferenceTopicEntry.tsx

```diff
@@ -72,10 +73,27 @@ const ReferenceTopicEntry = ({
           ...docStyles,
         }}
       >
-        <DocumentationHeading name={entry.name} isV2Only={isV2Only(entry)} />
+        <HStack justifyContent="space-between" flexWrap="nowrap">
+          <DocumentationHeading name={entry.name} isV2Only={isV2Only(entry)} />
+          <ShowMoreButton
+            isBrief
+            onClick={disclosure.onToggle}
+            isOpen={disclosure.isOpen}
+          />
+        </HStack>
 
+        {!disclosure.isOpen && (
+          <Text noOfLines={1} as="div">
+            <DocumentationContent
+              content={content && content.length > 0 ? [content[0]] : []}
```

Comment:
I guess it's possible the first block won't be text. We should probably check this and skip it in that case (similarly we can skip in the empty/missing content cases rather than passing `[]`).